relay-server/src/actors/global_config.rs

```diff
@@ -256,8 +265,12 @@ impl GlobalConfigService {
         match result {
             Ok(Ok(config)) => {
                 let mut success = false;
+                // Older relays won't send a global status, in that case, we will pretend like the
+                // default global config is an up to date one, because that was the old behaviour.
+                let is_ready = config.global_status.map_or(true, |stat| stat.is_ready());
```

Comment:
nit: Should we rename `config` to something like `response` here? Just to make it extra clear where it comes from.